.gitignore

```diff
@@ -44,4 +44,9 @@ Thumbs.db
 *.jar
 *.rar
 *.tar
-*.zip
\ No newline at end of file
+*.zip
+
+# Personal files
```

Comment:
These should either be in your global gitignore, remove from stage or these files shouldn't be in the folder you checked out. Or maybee we should just add a scripts/custom/private folder in which we could place these kind of files.

Reply:
I'm thinking that global gitignore should be the way to go. I currently do not have one.